channels/chan_simpleusb.c

```diff
@@ -906,6 +908,35 @@ static void *hidthread(void *arg)
 		 * found device.
 		 */
 		ast_radio_time(&o->lasthidtime);
+		/* If configuration has a serial number defined, find the device */
+		if (strlen(o->serial) > 0)
+		{
+			ast_log(LOG_NOTICE, "Checking for USB device with serial %s\n", o->serial);
+			int index = 0;
+			char *index_devstr = NULL;
+			for(;;)
+			{
+				index_devstr = ast_radio_usb_get_devstr(index);
+				if (ast_strlen_zero(index_devstr)) {
+					/* No more devices, so break out of the loop */
+					ast_log(LOG_NOTICE, "USB Device with serial %s not found.\n", o->serial);
+					break;
+				}
+				/* Go through the list of usb devices, and get the serial numbers */
+				if (ast_radio_get_usb_serial(index_devstr, serial) == 0) continue;
```

Comment:
The app_rpt project follows the asterisk coding standards where single/standalone statements are not allowed.   This should be :
```
if (ast_radio_get_usb_serial(index_devstr, serial) == 0) {
    continue;
}
```